msteams-platform/bots/how-to/conversations/conversation-basics.md

```diff
@@ -373,11 +398,36 @@ Use the following code to send a simple adaptive card:
 
 To know more about cards and cards in bots, see [cards documentation](~/task-modules-and-cards/what-are-cards.md).
 
+The next section provides status code responses for errors generated from Bot APIs.
+
+## Status code responses
+
+The following image shows a sample 403 status code response:
+
+![Sample 403 response](~/assets/images/sample403response.png)
+
+Following are the status codes and their error code and message values:
+
```

Comment:
Could you please help to make following changes?
1. Swap the position of description and error code columns.
2. Add code and message on two separate lines and update the formatting
**Code**: `ConversationBlockedByUser` 
**Message**: User blocked the conversation with the bot.